crates/optimism/node/src/node.rs

```diff
@@ -500,18 +500,29 @@ where
                 },
             );
 
-            // spawn the maintenance task
+            // spawn the main maintenance task
             ctx.task_executor().spawn_critical(
                 "txpool maintenance task",
                 reth_transaction_pool::maintain::maintain_transaction_pool_future(
                     client,
-                    pool,
+                    pool.clone(),
                     chain_events,
                     ctx.task_executor().clone(),
                     Default::default(),
                 ),
             );
             debug!(target: "reth::cli", "Spawned txpool maintenance task");
+
+            // spawn the Op txpool maintenance task
+            let chain_events = ctx.provider().canonical_state_stream();
+            ctx.task_executor().spawn_critical(
+                "Op txpool maintenance task",
+                reth_optimism_txpool::maintain::maintain_transaction_pool_future(
+                    pool,
+                    chain_events,
+                ),
+            );
```

Comment:
ideally this also has access to the enable conditional setting so we can spawn this based on that, I believe we can pass this setting down